### PR TITLE
Load assets

### PR DESCRIPTION
**Pull request by KevinElberger**
Loads the tilemap, adjusts the Electron window view, and removes unnecessary scaling done in boot.js

### CODE REVIEW

File: app/sass/style.scss [hunk 1]
```diff
@@ -6,3 +6,15 @@ body {
     background-color: #333;
     margin: 0 auto;
 }
+
+canvas {
```
Comment (StevenElberger) [inline app/sass/style.scss]: what is this?

File: app/vendor/states/game.js [hunk 2]
```diff
@@ -6,7 +6,19 @@ PhaserBp.Game = function (game) {
 PhaserBp.Game.prototype = {
 
   create() {
-    this.add.sprite(400 - 32, 300 - 32, 'renderTest');
+    this.GRAVITY = 130;
```
Comment (StevenElberger) [inline app/vendor/states/game.js]: if we only use this in this fn, why not use `const`?

File: app/vendor/main.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 // INIT FUNCTION 
 window.onload = function () {
-  const game = new Phaser.Game(800, 600, Phaser.AUTO, 'phaser-stage');
+  const game = new Phaser.Game(320, 96, Phaser.AUTO, 'phaser-stage');
```
Comment (StevenElberger) [inline app/vendor/main.js]: these seem like magic numbers, maybe extract into `const` vars so we can make this line more readable

File: app/sass/style.scss (hunk 1 above)
Comment (KevinElberger) [reply app/sass/style.scss]: This tells the browser how to scale images by specifying which algorithm to use. See [https://developer.mozilla.org/en-US/docs/Web/CSS/image-rendering](here) for details.

File: app/vendor/states/game.js (hunk 2 above)
Comment (KevinElberger) [reply app/vendor/states/game.js]: I agree with this, however we might want to break the physics portion of this outside of the create() method. Either way, I'll change this for now.

**Pull request merged**